scripts/aio_cleanup.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+import datetime
+import dateutil.parser
+import os
+
+from novaclient import client
+
+SLAVE_NAME = os.environ.get('SLAVE_NAME', 'Jenkins-OM-AIO')
+NUM_HRS = os.environ.get('NUM_HRS', 6)
```

Comment:
On line 26, I've wrapped the `NUM_HRS` variable as an integer. That should help resolve this.